### PR TITLE
Add version on request header.

### PR DESCRIPTION
**Pull request by rizakii**
related : KiiPlatform/KiiThingSDK-Embedded#107

### CODE REVIEW

File: kii_thing_if.c [hunk 1]
```diff
@@ -131,6 +131,10 @@ static kii_bool_t prv_init_kii_thing_if(
     kii_thing_if->command_handler.kii_core.http_context.buffer_size =
         command_handler_resource->buffer_size;
 
+        kii_sprintf(kii_thing_if->command_handler.kii_core,
```
Comment (dairyo) [inline kii_thing_if.c]: I think `kii_core` should be `kii_core.sdk_info`. What do you think?

Indent is wrong. Could you fix this?

**Comment by dairyo** [comment]
@rizakii I commented. Could you check this?

**Comment by satoshikumano** [comment]
@rizakii 
The header value format is
sn={sdk name};sv={sdk version};pv={platform version}

pv is optional. Since we don't know platform we'll omit pv.
sn={sdk name};sv={sdk version}

Let's define sdk name as 'tic' (thing if c).

**Comment by dairyo** [comment]
Related issue is changed to related : KiiPlatform/KiiThingSDK-Embedded#108

**Comment by satoshikumano** [comment]
Done in another PR.

**Pull request closed without merging**